x/fswap/types/errors.go

```diff
@@ -8,5 +8,7 @@ import (
 
 // x/fswap module sentinel errors
 var (
-	ErrSample = sdkerrors.Register(ModuleName, 1100, "sample error")
+	ErrParamsNotFound                 = sdkerrors.Register(ModuleName, 1100, "params does not exist")
+	ErrSwappedNotFound                = sdkerrors.Register(ModuleName, 1101, "swapped does not exist")
+	ErrSwappableNewCoinAmountNotFound = sdkerrors.Register(ModuleName, 1102, "swappable new coin amount does not exist")
 )
```

Comment:
I've found any specific reason for this sentinel errors, why don't we use general one `sdkerrors.ErrNotFound`?

Reply:
I would like to know which part cannot found